lib/db.js

```diff
@@ -113,7 +113,7 @@ const legalOptionNames = [
  * @param {object} [options.pkFactory] A primary key factory object for generation of custom _id keys.
  * @param {object} [options.promiseLibrary] A Promise library class the application wishes to use such as Bluebird, must be ES6 compatible
  * @param {object} [options.readConcern] Specify a read concern for the collection. (only MongoDB 3.2 or higher supported)
- * @param {object} [options.readConcern.level='local'] Specify a read concern level for the collection operations, one of [local|majority]. (only MongoDB 3.2 or higher supported)
+ * @param {object} [options.readConcern.level='local'] Specify a read concern level for the collection operations, one of ['local'|'available'|'majority'|'linearizable'|'snapshot']. (only MongoDB 3.2 or higher supported)
```

Comment:
can we use jsdoc a little more here maybe? Looking at this [comment](https://github.com/jsdoc3/jsdoc/issues/1065#issuecomment-454439595), it seems like we could do:
```
@typedef {'local' | 'available' | 'majority' | 'linearizable' | 'snapshot' } ReadConcernLevel

....

@param {ReadConcernLevel} [options.readConcern.level='local'] Specify a read concern level for the collection operations. (only MongoDB 3.2 or higher supported)
```

Reply:
@mbroadst updated